core/import.go

```diff
@@ -193,18 +189,14 @@ listingLoop:
 			pos, ok = fields["image_urls"]
 			if ok {
 				listing.Item.Images = []*pb.Listing_Item_Image{}
-				imageUrls := stringSlicePool.Get().([]string)
-				imageUrls = []string{}
-				imageUrls = strings.Split(record[pos], ",")
```

Comment:
I have no idea why or what is going on here with this triple-assignment attempt. What purpose does this even serve?

Reply:
That's a great question. My guess is the thought was the first one gets the memory location from the pool, the second is clearing it out, and the 3rd is setting it to desired value?

Regardless, I think the new code is more correct.